packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -48,7 +48,9 @@ const JurisdictionScreen = ({
     return (
         <div
             className={twMerge(
-                'flex flex-col h-auto w-[85vw] items-center justify-center my-auto mx-30 sm:h-[75vh] transition-all ease-in duration-[0.6s]',
+                `flex flex-col ${
+                    marketType === MarketType.FINANCIAL ? 'w-[1200px] h-[642px]' : 'w-[1040px] h-[592px]'
```

Comment:
Can we have one default width and height and do another condition below to overwrite it. Can follow the isDynamicLeverageVisible && approach below

Reply:
Will find a workaround that don't require the use of marketType or business terms